packages/n/niri/package.yml

```diff
@@ -0,0 +1,46 @@
+name       : niri
+version    : 25.05.1
+release    : 1
+source     :
+    - https://github.com/YaLTeR/niri/archive/refs/tags/v25.05.1.tar.gz : 2df192fa85adbb0eabcfddbd8aa812c3d3329a582a7e62212ea9fcddafa79b00
+homepage   : https://github.com/YaLTeR/niri
+license    : GPL-3.0-or-later
+component  : desktop
+summary    :
+    - session : Session files for Niri
+    - A scrollable-tiling Wayland compositor
+description:
+    - session : Session files for Niri.
+    - A scrollable-tiling Wayland compositor.
+builddeps  :
+    - pkgconfig(gbm)
+    - pkgconfig(libdisplay-info)
+    - pkgconfig(libinput)
+    - pkgconfig(libseat)
+    - pkgconfig(pangocairo)
+    - pkgconfig(xkbcommon)
+    - rust
+rundeps    :
+    - session :
+        - niri
+    - xwayland-satellite
```

Comment:
Let's at least add `xdg-desktop-portal-gtk`. We can think about whether it's necessary to add `xdg-desktop-portal-gnome` later.